VBA/Excel-VBA/articles/excel-performance-and-limit-improvements.md

```diff
@@ -1,247 +1,165 @@
+
+LAA has now been enabled for 32-bit versions of Excel 2013 and Excel 2016, and will minimize out-of-memory error messages. LAA doubles available virtual memory from 2 GB to 4 GB when using 64-bit Windows, and increases available virtual memory from 2 GB to 3 GB under 32-bit Windows.
 
-For more details see [LAA Capability Change for Excel](https://support.microsoft.com/en-ca/help/3160741/large-address-aware-capability-change-for-excel "LAA Capability Change for Excel") 
-To download a tool that shows how much virtual memory is available and how much is being used see [Excel Memory Checking Tool](https://fastexcel.wordpress.com/2016/11/27/excel-memory-checking-tool-using-laa-to-increase-useable-excel-memory/)
+For more information, see [LAA Capability Change for Excel](https://support.microsoft.com/en-ca/help/3160741/large-address-aware-capability-change-for-excel "LAA Capability Change for Excel"). 
 
-### Full Column References
-Previously, workbooks using large numbers of full column references and multiple worksheets, for example `=COUNTIF(Sheet2!A:A,Sheet3!A1)`, might use large amounts of CPU and memory when opened, or rows were deleted. 
-An improvement in Excel 2016 build 16.0.8212.1000 substantially reduces the memory and CPU used in these circumstances.
+To download a tool that shows how much virtual memory is available and how much is being used, see [Excel Memory Checking Tool](https://fastexcel.wordpress.com/2016/11/27/excel-memory-checking-tool-using-laa-to-increase-useable-excel-memory/).
+
+### Full column references
+Previously, workbooks using large numbers of full column references and multiple worksheets (for example `=COUNTIF(Sheet2!A:A,Sheet3!A1)`), might use large amounts of CPU and memory when opened, or rows were deleted. An improvement in Excel 2016 Build 16.0.8212.1000 substantially reduces the memory and CPU used in these circumstances.
  
-*My test on a workbook with 6 million formulas using full column references failed with an Out of Memory message at 4 GB of virtual memory with Excel 2013 LAA but only used 2 GB virtual memory with Excel 2016*.
+*My test on a workbook with 6 million formulas using full column references failed with an Out of Memory message at 4 GB of virtual memory with Excel 2013 LAA, but only used 2 GB virtual memory with Excel 2016*.
 
-### Structured References
-In some circumstances editing Excel Tables where formulas in the workbook use Structured References to the Table could be slow with Excel 2013 and previous versions. This led to the perception that Tables should not be used with large numbers of rows.
-Excel 2016 has now fixed this problem. 
+### Structured references
+In some circumstances, editing Excel tables where formulas in the workbook use structured references to the table could be slow with Excel 2013 and previous versions. This led to the perception that tables should not be used with large numbers of rows. Excel 2016 has now fixed this problem. 
```

Comment:
Change to "In Excel 2013 and earlier versions, editing tables... is slow. This issue no longer occurs in Excel 2016."